frontend/src/concepts/pipelines/content/createRun/RunPageFooter.tsx

```diff
@@ -49,11 +52,18 @@ const RunPageFooter: React.FC<RunPageFooterProps> = ({ data, contextPath }) => {
                 setSubmitting(true);
                 setError(null);
                 handleSubmit(data, api)
-                  .then((path) => {
-                    navigate({
-                      pathname: `${contextPath}/${path}`,
-                      search: runType ? `?${PipelineRunSearchParam.RunType}=${runType}` : '',
-                    });
+                  .then((resource) => {
+                    let detailsPath = isRunSchedule(resource)
+                      ? routePipelineRunJobDetails(resource.recurring_run_id)
+                      : routePipelineRunDetails(resource.run_id);
+
+                    if (isExperimentsAvailable && experimentId) {
+                      detailsPath = isRunSchedule(resource)
+                        ? resource.recurring_run_id
+                        : resource.run_id;
+                    }
+
+                    navigate(`${contextPath}/${detailsPath}`);
```

Comment:
This route probably should be in the routes folder so it's all together with the structure. Otherwise this is just what we had before routes with extra steps 🤔

Reply:
I'm confused. Is this a blocking comment? If so, where would you like me to place this as a function exactly and what would you suggest it be called?

Reply:
If we are ready to merge, perhaps not -- but this seems questionable given the intent to support an id properly in functions so we didn't concat them on the fly in components.

Reply:
Oh wow -- I'm mis-remembering what `contextPath` was... ignore me... this cannot be done via a function. it's putting to parts together.